app/src/main/java/com/example/user/epam_hw/FragmentDownloads.java

```diff
@@ -0,0 +1,38 @@
+package com.example.user.epam_hw;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+public class FragmentDownloads extends Fragment {
```

Comment:
Class name should be SomeNameTypeOfFrameworkClass: DownloadFragment but layouts has this type: type_of_framework_class_name_of_class - fragment_download, activity_main, view_profile etc